docker/README.md

```diff
@@ -127,6 +133,7 @@ You can specify the following environment variables when issuing `docker run` co
 
 | Variable | Type | Default | Function |
 | --- | --- | --- | --- |
+| `ADAPTER_NAME` | string |  | DB adapter name (use with `tinode/tinode-all` container only) |
```

Comment:
Maybe `STORE_USE_ADAPTER`? Namespacing to store more consistent with `MEDIA_HANDLER` (which is less than ideal itself, it should be named `MEDIA_USE_HANDLER`).

Reply:
BTW, it should not be a problem if it's used with any other docker.

Reply:
Switched to `STORE_USE_ADAPTER`.